### DISCUSSION TITLE
Checking all folders

### DISCUSSION THREAD

**Discussion by VlaK0r** [the-djmaze/snappymail#1370] [Q&A]
Hello! 
Thank you for your hard work!

Checking for new emails takes place only in the "Inbox". But I have other manually created folders where incoming messages are moved according to the rules. For some reason, automatic verification does not occur in them.
Please tell me how to fix it?

**Comment by the-djmaze** [comment]
Currently not.
Every time a message arrives in a folder, the notification would pop up.
So.when you move messages, this also happens.

First we need to figure out a way to prevent notifications for messages that are moved.
